students/Snayxts/module-2-html/exercises/zdk6/zvuk.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Monthly Top 5 - Free Music Archive</title>
+</head>
+<body>
+    <h1>Free Music Archive</h1>
+    <br>
+
+    <h2>Monthly Top 5 (December 2019)</h2>
+    <h3>#1</h3>
+    <ul>
+        <li>Artist: Broke For Free</li>
+        <li>Track: Night Owl</li>
+        <li>Album: Directionless EP</li>
+        <li>Genre: Electronic, Minimal</li>
+    </ul>
+
+    <audio controls>
+        <source src="Broke_For_Free_-_01_-_Night_Owl.mp3" type="audio/mp3">

```

Comment:
Preporučam koristiti fallback tekst kada se koristi `<audio>` tag. Na taj način možemo informirati posjetitelje s preglednicima bez podrške za `<audio>` element da tu postoji određeni sadržaj.